src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -291,6 +294,78 @@ public static void TestTryParseExactA()
         Assert.Equal(s, actual);
     }
 
+    [Fact]
+    [ActiveIssue(846, PlatformID.AnyUnix)]
+    public static void TestGetDateTimeFormats()
+    {
+        var savedCulture = CultureInfo.CurrentCulture;
+
+        try
+        {
+            CultureInfo.CurrentCulture = new CultureInfo("en-US");
+            DateTime july28 = new DateTime(2009, 7, 28, 5, 23, 15);
+
+            List<string> actualJuly28Formats = july28.GetDateTimeFormats().ToList();
+
+            Assert.Equal(expectedJuly28Formats.OrderBy(t => t), actualJuly28Formats.OrderBy(t => t));
```

Comment:
please change any test doing this pattern. we cannot assume the date/time format will never change for any culture